buffer/src/main/java/io/netty/buffer/PoolThreadCache.java

```diff
@@ -245,10 +245,8 @@ void free(boolean finalizer) {
                     free(smallSubPageHeapCaches, finalizer) +
                     free(normalHeapCaches, finalizer);
 
-            if (numFreed > 0 && logger.isDebugEnabled()) {
-                logger.debug("Freed {} thread-local buffer(s) from thread: {}", numFreed,
-                        Thread.currentThread().getName());
-            }
+            logger.debug("Freed {} thread-local buffer(s) from thread: {}", numFreed,
```

Comment:
what happened to `numFreed > 0`

Reply:
Thanks for reviewing @johnou !

It seemed weird to me that it is logging only when `numFreed > 0`.  The absence of logging indicates, in a round-about-way, that `numFreed <= 0`.  It is a debug log statement.  It should simply print what is encountered.  It seems like this would otherwise require a unit test to enforce that this logs only when this condition is met.  It is more simple and straightforward to simply do the logging without conditions.

Reply:
> It seemed weird to me that it is logging only when `numFreed > 0`

Why is that weird? The intention was clearly to log when buffers are getting freed, which doesn't imply it is desired/useful to log every time this line of code is passed.

Reply:
I call it weird, as in unexpected, because it seems like someone put that in there to debug a  specific issue.  It is more helpful that it simply always logs the number of buffer freed, even if it is zero.  Removing zero buffers may just as interesting as freeing any other number of buffers.  In fact, removing 0 buffers may be even more interesting than most other values.  If the logging is found to be too verbose, it is better to lower the logging level.  Removing this check makes the logging more consistent ("Hey! Why did I not get a log message this time!?") and has the added benefit of removing code.